src/helpers/Utils.cpp

```diff
@@ -54,20 +54,20 @@ GetFileExtension(const std::string filename)
 }
 
 
-void
+status_t
 GetVectorIcon(const std::string icon, BBitmap* bitmap)
 {
 	if(bitmap == nullptr)
```

Comment:
Could you add a space after if and before parenthesis? I know it was here before, but since you are touching this code...?